src/main/java/bowling/domain/FrameNumber.java

```diff
@@ -41,11 +47,11 @@ public int hashCode() {
 
     private void verifyFrameNumber(int frameNumber) {
         if (frameNumber > LAST_FRAME_NUMBER) {
-            throw new IllegalArgumentException("잘못된 프레임 번호");
+            throw new WrongFrameNumberException("잘못된 프레임 번호");
         }
 
         if (frameNumber < FIRST_FRAME_NUMBER) {
-            throw new IllegalArgumentException("잘못된 프레임 번호");
+            throw new WrongFrameNumberException("잘못된 프레임 번호");
```

Comment:
```suggestion
            throw new WrongFrameNumberException();
```

제가 의도했던 것은 WrongFrameNumberException 안에 해당하는 메시지를 같이 갖도록 하는것이었습니다.
다른 커스텀 Exception 에 대해서도 마찬가지입니닷.
취향 차이 이므로, 꼭 반드시 반영해 주실 필요는 없습니다.

```java
public class WrongFrameNumberException extends IllegalArgumentException {
    public WrongFrameNumberException () {
        super("잘못된 프레임 번호");
    }
}
```